Adding analytics for nxo and bumping native version (#599)
braintree/braintree_android: CHANGELOG.md

# Braintree Android SDK Release Notes

## unreleased

> * PayPalNativeCheckout

Reviewer: ```suggestion
* PayPalNativeCheckout (BETA)
```

  * Bumping native-checkout version to 0.8.2

>   * Bugfix to support logging out of session when client ID changes

Reviewer 2: Is this the same bugfix as mentioned on iOS [here](https://github.com/braintree/braintree_ios/pull/872/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4ed)?

Author: yup thats corect, I'll match up the text with theirs

# Braintree Android SDK Release Notes

> ## Unreleased

Reviewer: ```suggestion
## unreleased
```

* PayPalNativeCheckout

>   * Bumping native-checkout version to 0.8.1

Reviewer: Based on the `build.gradle` diff should this be `0.8.2`?

Author: ahh yeah thats correct let me update

>   * Bugfix to support logging out when client id's switch

Reviewer: ```suggestion
  * Bugfix to support logging out of session when client ID changes
```

>   * Bugfix for auth-sdk obfuscation

Reviewer: What specifically is being fixes with the obfuscation of the auth SDK? Is this entry needed for public facing merchants?

Author: it was a duplicate class file, its actually probably not needed to declare here. We saw an issue with LMVH about it and something broke with obfuscation but i'll remove this entry
